src/server/scripts/Outland/zone_shadowmoon_valley.cpp

```diff
@@ -514,6 +514,8 @@ class npc_dragonmaw_peon : public CreatureScript
             Initialize();
         }
 
+        void EnterCombat(Unit* /*who*/) override { }
```

Comment:
Nothing in particular. I just added it based on it being there in similar scripts. I will remove it.